src/read.rs

```diff
@@ -521,6 +521,13 @@ impl<R> WavReader<R> where R: io::Read {
     pub fn len(&self) -> u32 {
         self.num_samples
     }
+
+    /// Destroys the `WavReader` and returns the underlying reader.
+    ///
+    /// The amount of data that has been read from the reader is undefined.
```

Comment:
> The amount of data that has been read from the reader is undefined.

Can you please remove this? The amount of data that has been read is not undefined, it depends on how many samples were read. There is no internal buffering; if you use `WavReader::open` to construct the reader this will return the `BufReader`.